### PR TITLE
Bump chart version

### PR DESCRIPTION
**Pull request by TwoStone**

### CODE REVIEW

File: chart/Chart.yaml [hunk 1]
```diff
@@ -22,3 +22,8 @@ annotations:
       links:
```
Comment (Vad1mo) [inline chart/Chart.yaml:22]: the file should only contain current changes, as the complete history is assembled in artifacthub.io

**Review by Vad1mo — CHANGES_REQUESTED** [review]

**Comment by Vad1mo** [comment]
I also recommend linting the yaml file with https://artifacthub.io/docs/topics/cli/.

File: chart/Chart.yaml (hunk 1 above)
Comment (TwoStone) [reply chart/Chart.yaml:22]: Okay, I hadn't realised that.

**Comment by TwoStone** [comment]
> I also recommend linting the yaml file with https://artifacthub.io/docs/topics/cli/.

Thanks for the feedback. I have done so and would be delighted to receive another review. Thank you in advance

**Pull request merged**